stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1862,7 +1860,7 @@ internal constructor(
      */
     @CheckResult
     public fun stopWatching(channelType: String, channelId: String): Call<Unit> {
-        return callPostponeHelper.postponeCall { api.stopWatching(channelType, channelId) }
+        return api.postponeCall { api.stopWatching(channelType, channelId) }
```

Comment:
As far as it is always postponed, it can be handled within `api.stopWatching()` method